File: LimsUI/Models/UIModels/StandardData.cs
using System.Text.Json.Serialization;

namespace LimsUI.Models.UIModels
{
    public class StandardData
    {
        [JsonPropertyName("measValue")]
        public float MeasureValue { get; set; }

        [JsonPropertyName("concentration")]
        public float Concentration { get; set; }
    }
}

File: lims-process/src/main/java/com/example/workflow/Models/StandardData.java
package com.example.workflow.Models;

public class StandardData {

    private int pos;
    private float concentration;
    private float measValue;

    public StandardData() {
    }

    public StandardData(int pos, float concentration, float value) {
        this.pos = pos;
        this.concentration = concentration;
        this.measValue = value;
    }

    public int getPos() {
        return pos;
    }

    public float getConcentration() {
        return concentration;
    }

    public float getMeasValue() {
        return measValue;
    }
}
